### ISSUE TITLE
Do not run fix.sh

### ISSUE THREAD

**Issue by fii34** [Foggalong/hardcode-fixer#141]
The usual way fix.sh not start - command not found (root also)
I was able to run fix.sh only after this command in Terminal:

bash fix.sh

and once given the right to read your name in the record /tmp/tofix.csv

**Comment by Foggalong** [comment]
I have no idea what you're saying sorry :( what language do you speak? try reporting in your native country and I'll see if I can work it out

**Comment by fii34** [comment]
I do not know English. Translate google.
See screenshot.

http://i.imgur.com/durVPqn.png

**Comment by Foggalong** [comment]
Are you using "zsh", " fish", or some other shell?

**Comment by fii34** [comment]
Maybe... I'm not proffesional.

I am now worried about another matter. How do you define the properties Launcher?
I'm trying to fix your icons in your CSV.

These work:

3D Acceleration,driconf,/usr/share/driconf/driconf-icon.png,driconf-icon
Side-scrolling game,abe,/usr/share/pixmaps/abe.xpm,abe
AClock,AClock,/usr/lib/GNUstep/Applications/AClock.app/Resources/AClock.tiff,aclock.app
Aconnectgui,aconnectgui,/usr/share/pixmaps/aconnectgui.xpm,aconnectgui
Alsamixergui,alsamixergui,/usr/share/pixmaps/alsamixergui.xpm,aconnectgui
Advene Video annotation,advene,/usr/share/advene/pixmaps/icon_advene32.png,advene
amSynth,amsynth,/usr/share/pixmaps/amsynth.xpm,amsynth
apvlv,apvlv,/usr/share/pixmaps/apvlv/apvlv.xpm,apvlv
Artemis,artemis,/usr/share/pixmaps/artemis.xpm,artemis
Athena,athena,/usr/share/pixmaps/athena.xpm,athena
Auralquiz,auralquiz,/usr/share/pixmaps/auralquiz.png,auralquiz
Auto Multiple Choice,auto-multiple-choice,/usr/share/auto-multiple-choice/icons/auto-multiple-choice.svg,auto-multiple-choice
ayttm,ayttm,/usr/share/pixmaps/ayttm.png,ayttm
Bacula Monitor,bacula-tray-monitor,/usr/share/pixmaps/bacula-tray-monitor.xpm,bacula-tray-monitor
Basenji,basenji,/usr/share/basenji/basenji.svg,basenji
BASIC-256,basic256,/usr/share/pixmaps/basic256.xpm,basic256
Biniax-2,biniax2,/usr/share/games/biniax2/graphics/logo.png,biniax2
BitPim,bitpim,/usr/share/bitpim/resources/bitpim.ico,bitpim
BlockOut II,blockout2,/usr/share/games/blockout2/icon.xpm,blockout2
BlueWho,bluewho,/usr/share/bluewho/data/bluewho.svg,bluewho
bpython,bpython,/usr/share/pixmaps/python.xpm,bpython
Brewtarget,brewtarget,/usr/share/icons/brewtarget/brewtarget_icon.svg,brewtarget_icon
Briquolo,briquolo,/usr/share/pixmaps/briquolo.svg,briquolo
Cain,cain,/usr/share/cain/gui/icons/cain.png,cain
Camera Monitor,cameramonitor,/usr/share/cameramonitor/cameramonitor.png,cameramonitor
Cecilia,cecilia,/usr/share/pixmaps/cecilia.png,cecilia
Bacula Administration Tool,bat,/usr/share/pixmaps/bat_icon.png,bat_icon
Code of Conduct Signing Assistant,codeOfConductSigningAssistant,/usr/share/icons/hicolor/scalable/apps/codeOfConductSigningAssistant.svg,codeOfConductSigningAssistant
CodFis,codfis,/usr/share/pixmaps/codfis.xpm,codfis
ColorCode,colorcode,/usr/share/pixmaps/colorcode32.xpm,colorcode32
Charmap,Charmap,/usr/lib/GNUstep/Applications/Charmap.app/Resources/Charmap.png,Charmap
Ardentryst,Ardentryst,/usr/share/pixmaps/ardentryst.png,ardentryst
Ardour Digital Audio Workstation,ardour,/usr/share/ardour2/icons/ardour_icon_22px.png,ardour
Camera,Camera,/usr/lib/GNUstep/Applications/Camera.app/Resources/Camera.png,Camera
AltOS UI,altos,/usr/share/pixmaps/altusmetrum.xpm,altusmetrum
colorname,colorname,/usr/share/pixmaps/colorname.xpm,colorname
Configure-Debian,configure-debian,/usr/share/pixmaps/configure-debian-48x48.png,configure-debian
ConvertAmicasJPEG2000FilesetToDicom,ConvertAmicasJPEG2000FilesetToDicom,/usr/share/pixelmed/icons/ConvertAmicasJPEG2000FilesetToDicom.png,ConvertAmicasJPEG2000FilesetToDicom
Cynthiune,Cynthiune,/usr/lib/GNUstep/Applications/Cynthiune.app/Resources/Cynthiune.tiff,cynthiune.app
dc-qt,dc-qt,/usr/share/pixmaps/dc-qt.xpm,dc-qt

These do not work:

Composite,composite,composite32x32,composite
ClusterSSH,ClusterSSH,/usr/share/pixmaps/clusterssh-32x32.xpm,clusterssh
Classrooom control,controlaula,controlaula,/usr/share/pixmaps/controlaula.png,controlaula
CLAM Prototyper,prototyper,/usr/share/pixmaps/clam-prototyper.xpm,clam-prototyper
CLAM Network Editor,networkeditor,/usr/share/pixmaps/clam-networkeditor.xpm,clam-networkeditor
CLAM Chordata,Chordata,/usr/share/pixmaps/clam-chordata.xpm,clam-chordata
Camera Monitor Preferences,Camera Monitor Preferences,/usr/share/cameramonitor/cameramonitor.png,cameramonitor
Cenon,cenon.app,/usr/lib/GNUstep/Applications/Cenon.app/Resources/Cenon.tiff,cenon.app
Canorus,Canorus,/usr/share/canorus/images/clogosm.png,clogosm
Brp-pacu,Brp-pacu,brp-pacu_32x32,brp-pacu
BitTorrent Download Client,BitTorrent Download Client,download,bittorrent-download-client
Battery Charge Graph,Battery Charge Graph,/usr/share/battery-graph/pixmaps/battery-stats.xpm,battery-stats
Battery Monitor (batmon),batmon,/usr/lib/GNUstep/Applications/batmon.app/Resources/Batmon_icon.tiff,Batmon_icon
Aliki (with ALSA support),Aliki,aliki_32x32,aliki
Aliki (with JACK support),Aliki,aliki_32x32,aliki
Adun,adun,/usr/lib/GNUstep/Applications/UL.app/Resources/icon.tiff,Adun
Amoebax,Amoebax,amoebax.svg,amoebax
Angband (SDL),Angband,/usr/share/angband/xtra/icon/att-32.png,att
Angband (X11),Angband,/usr/share/angband/xtra/icon/att-32.png,att
Angband (GTK),Angband,/usr/share/angband/xtra/icon/att-32.png,att
Aghermann,Aghermann,/usr/share/aghermann/aghermann.png,aghermann

Icons I created and see, but the script does not do the job. It seems to me that the problem is not Correctly name for Launcher? 

What are the error?

**Comment by Foggalong** [comment]
What distro do you use?

The launcher is probably just a case of you misspelling the name or mixing up the variables

**Comment by fii34** [comment]
XFCE + Lubuntu + MATE (weak computer)

I installed all the programs through lubuntu-software-center
It is about 2800 programs. This themes has almost all the icons:

https://yadi.sk/d/fw4aJ_f4gYiXp

I want to register in the rest of the script 320 icons. You have the ability to check the script for errors?

Icons easily search through mate-search-tool apps you need to specify the themes and insert the name of the icon (it is the last in the row for each csv).

To see the icon found it can simply drag and drop to the desktop of the mate-search-tool


It loads a script or not immediately visible in the open folder/ usr/share/applications

If you are interested, I can choose .desktop files do not appear new icons, compress them and put the link here.

The .desktop files have a full path to the icon and it is easy to write instructions for the csv.

Unfortunately, I can not understand why some rules are triggered, while others do not. Again, almost all icons in the themes is. It is necessary to properly prescribe them the way

translate google

**Comment by fii34** [comment]
Fix...

These work:

3D Acceleration,driconf,/usr/share/driconf/driconf-icon.png,driconf-icon
Side-scrolling game,abe,/usr/share/pixmaps/abe.xpm,abe
AClock,AClock,/usr/lib/GNUstep/Applications/AClock.app/Resources/AClock.tiff,aclock.app
Aconnectgui,aconnectgui,/usr/share/pixmaps/aconnectgui.xpm,aconnectgui
Alsamixergui,alsamixergui,/usr/share/pixmaps/alsamixergui.xpm,aconnectgui
Advene Video annotation,advene,/usr/share/advene/pixmaps/icon_advene32.png,advene
amSynth,amsynth,/usr/share/pixmaps/amsynth.xpm,amsynth
apvlv,apvlv,/usr/share/pixmaps/apvlv/apvlv.xpm,apvlv
Artemis,artemis,/usr/share/pixmaps/artemis.xpm,artemis
Athena,athena,/usr/share/pixmaps/athena.xpm,athena
Auralquiz,auralquiz,/usr/share/pixmaps/auralquiz.png,auralquiz
Auto Multiple Choice,auto-multiple-choice,/usr/share/auto-multiple-choice/icons/auto-multiple-choice.svg,auto-multiple-choice
ayttm,ayttm,/usr/share/pixmaps/ayttm.png,ayttm
Bacula Monitor,bacula-tray-monitor,/usr/share/pixmaps/bacula-tray-monitor.xpm,bacula-tray-monitor
Basenji,basenji,/usr/share/basenji/basenji.svg,basenji
BASIC-256,basic256,/usr/share/pixmaps/basic256.xpm,basic256
Biniax-2,biniax2,/usr/share/games/biniax2/graphics/logo.png,biniax2
BitPim,bitpim,/usr/share/bitpim/resources/bitpim.ico,bitpim
BlockOut II,blockout2,/usr/share/games/blockout2/icon.xpm,blockout2
BlueWho,bluewho,/usr/share/bluewho/data/bluewho.svg,bluewho
bpython,bpython,/usr/share/pixmaps/python.xpm,bpython
Brewtarget,brewtarget,/usr/share/icons/brewtarget/brewtarget_icon.svg,brewtarget_icon
Briquolo,briquolo,/usr/share/pixmaps/briquolo.svg,briquolo
Cain,cain,/usr/share/cain/gui/icons/cain.png,cain
Camera Monitor,cameramonitor,/usr/share/cameramonitor/cameramonitor.png,cameramonitor
Cecilia,cecilia,/usr/share/pixmaps/cecilia.png,cecilia
Bacula Administration Tool,bat,/usr/share/pixmaps/bat_icon.png,bat_icon
Code of Conduct Signing Assistant,codeOfConductSigningAssistant,/usr/share/icons/hicolor/scalable/apps/codeOfConductSigningAssistant.svg,codeOfConductSigningAssistant
CodFis,codfis,/usr/share/pixmaps/codfis.xpm,codfis
ColorCode,colorcode,/usr/share/pixmaps/colorcode32.xpm,colorcode32
Charmap,Charmap,/usr/lib/GNUstep/Applications/Charmap.app/Resources/Charmap.png,Charmap
Ardentryst,Ardentryst,/usr/share/pixmaps/ardentryst.png,ardentryst
Ardour Digital Audio Workstation,ardour,/usr/share/ardour2/icons/ardour_icon_22px.png,ardour
Camera,Camera,/usr/lib/GNUstep/Applications/Camera.app/Resources/Camera.png,Camera
AltOS UI,altos,/usr/share/pixmaps/altusmetrum.xpm,altusmetrum
colorname,colorname,/usr/share/pixmaps/colorname.xpm,colorname
Configure-Debian,configure-debian,/usr/share/pixmaps/configure-debian-48x48.png,configure-debian
ConvertAmicasJPEG2000FilesetToDicom,ConvertAmicasJPEG2000FilesetToDicom,/usr/share/pixelmed/icons/ConvertAmicasJPEG2000FilesetToDicom.png,ConvertAmicasJPEG2000FilesetToDicom
Cynthiune,Cynthiune,/usr/lib/GNUstep/Applications/Cynthiune.app/Resources/Cynthiune.tiff,cynthiune.app
dc-qt,dc-qt,/usr/share/pixmaps/dc-qt.xpm,dc-qt
DicomCleaner,DicomCleaner,/usr/share/pixelmed/icons/DicomCleaner.png,DicomCleaner
DicomImageBlackout,DicomImageBlackout,/usr/share/pixelmed/icons/DicomImageBlackout.png,DicomImageBlackout
DicomImageViewer,DicomImageViewer,/usr/share/pixelmed/icons/DicomImageViewer.png,DicomImageViewer
Dino,dino,/usr/share/dino/head.png,head
DoseUtility,DoseUtility,/usr/share/pixelmed/icons/DoseUtility.png,DoseUtility
DVBcut,dvbcut,/usr/share/dvbcut/icons/dvbcut.svg,dvbcut
EdenMath,EdenMath,/usr/lib/GNUstep/Applications/EdenMath.app/Resources/EdenMath.tiff,EdenMath
EDFviewer,edfviewer,/usr/share/pixmaps/pymca.xpm,pymca
Edubuntu Menu Editor,menueditor,/usr/share/edubuntu-menueditor/media/edubuntu-icon.svg,edubuntu-icon
Edubuntu menu editor - Profile manager,profilemanager,/usr/share/edubuntu-menueditor/media/edubuntu-icon.svg,edubuntu-icon
Eiciel,eiciel,/usr/share/eiciel/img/icon_eiciel_24.png,icon_eiciel_24
Elementsinfo,elementsinfo,/usr/share/pixmaps/pymca.xpm,pymca
emelFM2,emelfm2,emelfm2/emelfm2_48.png,emelfm2_48
emma,emma,/usr/share/emma/icons/emma.png,emma
Encuentro,encuentro,/usr/share/encuentro/encuentro/logos/icon-32.png,encuentro
Engauge-digitizer,engauge-digitizer,/usr/share/pixmaps/engauge-digitizer.xpm,engauge-digitizer
Epigrass,egigrass,/usr/share/pixmaps/egigrass.png,egigrass
E-Pyo,E-Pyo,/usr/share/pixmaps/E-Pyo.xpm,E-Pyo
eToys,etoys,/usr/share/pixmaps/activity-etoys.xpm,activity-etoys
Excellent Bifurcation,excellent-bifurcation,/usr/share/pixmaps/excellent-bifurcation.xpm,excellent-bifurcation
expEYES,expeyes,/usr/share/icons/expeyes.png,expeyes
FGo!,fgo,/usr/share/pixmaps/fgo.png
Firewall Builder,fwbuilder,/usr/share/pixmaps/fwbuilder/firewall_64.xpm,firewall_64
fldiff,fldiff,/usr/share/pixmaps/fldiff.xpm,fldiff
fotoxx,kornelix-fotoxx,/usr/share/fotoxx/icons/fotoxx.png,fotoxx
Fox,fox,/usr/share/pixmaps/fox.xpm,fox
fprint-demo,fprint-demo,/usr/share/pixmaps/fprint-demo.xpm,fprint-demo
FQTerm,fqterm,/usr/share/FQTerm/pic/fqterm_256x256.png,fqterm
Freebirth,freebirth,/usr/share/pixmaps/freebirth.xpm,freebirth
FreeCAD,freecad,/usr/share/freecad/freecad.xpm,freecad
Freedroid RPG,freedroidrpg,/usr/share/pixmaps/freedroidrpg.xpm,freedroidrpg
FreeMat,freemat,/usr/share/pixmaps/freemat/freemat.xpm,freemat
FreePOPs updater,freepops-updater-zenity,/usr/share/pixmaps/freepops-updater-zenity.svg,freepops-updater-zenity
Freevo,freevo,/usr/share/pixmaps/freevo.xpm,freevo
FreeWheeling,freewheeling,/usr/share/pixmaps/freewheeling.xpm,freewheeling
FTP,FTP,/usr/share/pixmaps/FTP.xpm,FTP
Full Kana+ Drill,kanadic-kata,/usr/share/pixmaps/kdrill.xpm,kdrill
Fyre,fyre,/usr/share/pixmaps/fyre.png,fyre
G3DViewer,g3dviewer,/usr/share/g3dviewer/glade/g3d48.png,g3d48
Garden of Coloured Lights,garden,/usr/share/pixmaps/garden.svg,garden
GaussSum,gausssum,/usr/share/pixmaps/gausssum.xpm,gausssum
GbgOffice,gbgoffice,/usr/share/pixmaps/gbgoffice-icon.png,gbgoffice-icon
gEcrit,gecrit,/usr/share/pixmaps/gEcrit.png,gEcrit
gExtractWinIcons,gextractwinicons,/usr/share/gextractwinicons/data/gextractwinicons.svg,gextractwinicons
GFM,gfm,/usr/share/gfm/pixmaps/gfm.xpm,gfm
Ghextris,ghextris,/usr/share/games/ghextris/ghextris.png,ghextris
Gip IP Address Calculator,gip,/usr/share/gip/calculator.png,calculator
Git Cola,git-cola-folder-handler,/usr/share/git-cola/icons/git.svg,git
Gitso,gitso,/usr/share/gitso/icon.ico,gitso
GkDebconf,gkdebconf,/usr/share/pixmaps/gkdebconf-icon.xpm,gkdebconf-icon
GLPeces,glpeces,/usr/share/pixmaps/glpeces.xpm,glpeces
gMobileMedia (Mobile Media Browser),gmobilemedia,/usr/share/gmobilemedia/ui/gmobilemedia_icon.svg,gmobilemedia_icon
GNOME ALSA Mixer,gnome-alsamixer,/usr/share/pixmaps/gnome-alsamixer/gnome-alsamixer-icon.png,gnome-alsamixer-icon
Gnome Catalog,gnomecatalog,/usr/share/gnomecatalog/pixmaps/gcatalog.png,gcatalog
GnomeKiSS,gnomekiss,/usr/share/pixmaps/gnomekiss/gnome-kiss.png,gnome-kiss
GNUbik,gnubik,/usr/share/pixmaps/gnubik.xpm,gnubik
GNUDoQ,gnudoq,/usr/share/pixmaps/gnudoq-icon.xpm,gnudoq-icon
GNUMail,GNUMail,/usr/share/GNUstep/GNUMail.app/GNUMail.tiff,gnumail.app
GNUSim8085,GNUSim8085,/usr/share/icons/hicolor/scalable/apps/gnusim8085.svg,gnusim8085
Google Drive,grive,/opt/thefanclub/grive-tools/GoogleDrive.png,GoogleDrive
Google Drive Indicator,grive-indicator/opt/thefanclub/grive-tools/GoogleDrive.png,GoogleDrive
Gorm,Gorm,/usr/lib/GNUstep/Applications/Gorm.app/Resources/Gorm.tiff,gorm.app
gResistor,gresistor,/usr/share/gresistor/icon.png,gresistor
Gridlock,Gridlock,/usr/lib/GNUstep/Applications/Gridlock.app/Resources/Gridlock.tiff,Gridlock
ClusterSSH,clusterssh,/usr/share/pixmaps/clusterssh-32x32.xpm,clusterssh
Camera Monitor Preferences,cameramonitor-prefs,/usr/share/cameramonitor/cameramonitor.png,cameramonitor
Cenon,Cenon,/usr/lib/GNUstep/Applications/Cenon.app/Resources/Cenon.tiff,cenon.app
Canorus,canorus,/usr/share/canorus/images/clogosm.png,clogosm
Adun,UL,/usr/lib/GNUstep/Applications/UL.app/Resources/icon.tiff,Adun
Amoebax,amoebax,amoebax.svg,amoebax
Angband (SDL),angband-sdl,/usr/share/angband/xtra/icon/att-32.png,att
Angband (X11),angband-x11,/usr/share/angband/xtra/icon/att-32.png,att
Angband (GTK),angband-gtk,/usr/share/angband/xtra/icon/att-32.png,att
Angry Drunken Dwarves,angrydd,/usr/share/games/angrydd/angrydd.png,angrydd
Classrooom control,controlaula,/usr/share/pixmaps/controlaula.png,controlaula


not work:

Desktop Flickr Organizer,FontBookSquareThumbnail,/usr/share/dfo/icons/FontBookSquareThumbnail.png,FontBookSquareThumbnail
Dictionary Reader,dict,/usr/lib/GNUstep/Applications/DictionaryReader.app/Resources/dict.png,dict
FSL 4.1,fsl-4.1,fsl-4.1.xpm,fsl-4.1
Gallery Uploader,gallery-uploader,gallery.svg,gallery
Git DAG,git-bag,/usr/share/git-cola/icons/git.svg,git
Glurp,glurp,glurp.svg,glurp
GNU Paint,gpaint,gpaint.svg,gpaint
Composite,composite,composite32x32,composite
CLAM Prototyper,Prototyper,/usr/share/pixmaps/clam-prototyper.xpm,clam-prototyper
CLAM Network Editor,NetworkEditor,/usr/share/pixmaps/clam-networkeditor.xpm,clam-networkeditor
CLAM Chordata,Chordata,/usr/share/pixmaps/clam-chordata.xpm,clam-chordata
Brp-pacu,brp-pacu,brp-pacu_32x32,brp-pacu
BitTorrent Download Client,gnome-btdownload,download,bittorrent-download-client
Battery Charge Graph,battery-stats,/usr/share/battery-graph/pixmaps/battery-stats.xpm,battery-stats
Battery Monitor (batmon),batmon,/usr/lib/GNUstep/Applications/batmon.app/Resources/Batmon_icon.tiff,batmon_icon
Aliki (with ALSA support),aliki-alsa,aliki_32x32,aliki
Aliki (with JACK support),aliki-jack,aliki_32x32,aliki
Aghermann,aghermann,/usr/share/aghermann/aghermann.png,aghermann

**Comment by fii34** [comment]
But there may be a conflict

/usr/share/applications

[Desktop Entry]
Exec=mintwelcome
Icon=mintInstall
Terminal=false
Type=Application
Encoding=UTF-8
Categories=GNOME;GTK;Settings;DesktopSettings;
NotShowIn=KDE;
StartupNotify=false
Name=Welcome Screen

Welcome Screen,mintWelcome,/usr/lib/linuxmint/mintWelcome/icons/logo.svg,mintInstall

/usr/share/applications/kde4

[Desktop Entry]
Exec=mintwelcome
Icon=/usr/lib/linuxmint/mintWelcome/icons/logo_kde.svg
Terminal=false
Type=Application
Encoding=UTF-8
OnlyShowIn=KDE;
Categories=System;
StartupNotify=false
Name=Welcome Screen

Welcome Screen,mintWelcome,/usr/lib/linuxmint/mintWelcome/icons/logo_kde.svg,kde

??

**Comment by Foggalong** [comment]
What happens when you run `echo $SHELL`?

**Comment by fii34** [comment]
/bin/bash

**Comment by bil-elmoussaoui** [comment]
I can't confirm that :/ @fii34 are you using `sudo ./fix.sh`?  using `sudo fix.sh` or `sudo sh fix.sh` won't work!

**Comment by fii34** [comment]
I found the cause of the problem. This security setting. If - fix.sh click on the right mouse button - Properties - Rights - tick in point - allows the execution of the file as a program, it works!

In the terminal there are questions, etc.

If you run
 
sudo bash fix.sh 

the script just works, without question, the selection and confirmation.

**Comment by Foggalong** [comment]
Huh, so this is a permissions problem! I think the script is already marked as executable in the repo so I'm not sure why this is happening. How did you download the script?

**Comment by fii34** [comment]
The latest version small bug.
the script does not work in /usr/share/applications/data/ and /usr/share/applications/kde4/
I corrected this way:

global_apps=("/usr/share/applications/" "/usr/share/applications/data/" "/usr/share/applications/kde4/" "/usr/local/share/applications/" "/usr/local/share/applications/kde4/")

everything works fine.

you - "/usr/local/share/applications/kde4" !!!
I have - "/usr/local/share/applications/kde4/"

**Comment by Foggalong** [comment]
@fii34 we'll fix that, but how about your permissions issue?

**Comment by fii34** [comment]
Script is run normally.

**Comment by fii34** [comment]
BIG bug: 

Can not create regular file «/home/m/.local/share/icons/hicolor/scalable/apps/lingot-icon»: No such file or directory
G: Fixing LinkChecker Commandline ...
G: Fixing LinkChecker Gui ...
L: Fixing Linphone ...
L: Fixing MailNag ...
cp: can not create regular file «/home/m/.local/share/icons/hicolor/scalable/apps/mailnag»: No such file or directory
L: Fixing Mandelbulber ...
G: Fixing Maps ...
G: Fixing Melange ...
G: Fixing mialmpick Version 0.2.10 ...
L: Fixing Netbeans IDE ...
G: Fixing NetEmul ...
G: Fixing NmapSI4 - Full mode ...
G: Fixing NmapSI4 - User mode ...
L: Fixing OCRFeeder ...
cp: can not create regular file «/home/m/.local/share/icons/hicolor/scalable/apps/ocrfeeder»: No such file or directory

I DO NOT HAVE  scalable/apps/


I have only IT


/home/m/.local/share/icons/hicolor/16x16
/home/m/.local/share/icons/hicolor/32x32
/home/m/.local/share/icons/hicolor/48x48
/home/m/.local/share/icons/hicolor/128x128

problem ?

local_scalable_icon="/home/${SUDO_USER:-$USER}/.local/share/icons/hicolor/scalable/apps/"

**Comment by fii34** [comment]
I DO NOT HAVE /usr/local/share/applications/ and /usr/local/share/applications/kde4/
Ubuntu 14.04 xfce, mate, kde,  
and LinuxMint 17 xfce, mate, kde

These folders NO

**Comment by Foggalong** [comment]
If the permissions issue is fixed, then I'll open a new issue for this.
